pennylane/templates/subroutines/adder.py

```diff
@@ -199,8 +200,8 @@ def compute_decomposition(
         else:
             qft_wires = work_wires[:1] + x_wires
             work_wire = work_wires[1:]
-        op_list.append(qml.QFT(qft_wires))
-        op_list.append(qml.PhaseAdder(k, qft_wires, mod, work_wire))
-        op_list.append(qml.adjoint(qml.QFT)(qft_wires))
+        op_list.append(QFT(qft_wires))
+        op_list.append(PhaseAdder(k, qft_wires, mod, work_wire))
+        op_list.append(adjoint(QFT)(qft_wires))
```

Comment:
```suggestion
        op_list.append(adjoint(QFT(qft_wires)))
```
while we're here.  This should be a safer syntax.